string event
string conditions
string format
string topic
string id
---
string result
